src/Plugin/VoteUpDownWidget/Alternate.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Drupal\vud\Plugin\VoteUpDownWidget;
+
+use Drupal\vud\Plugin\VoteUpDownWidgetBase;
+
+/**
+ * Plugin "alternate" of type VoteUpDownWidget.
+ *
+ * @VoteUpDownWidget(
```

Comment:
Now that I revisit this, I see that it is not clear even if you have reviewed D7 code, let me explain a little bit more about votes template: 
* it is an alternative visualization of the related vote information,
* originally this template showed the number of votes on node's links section,
* it was only used by `plain` widget,
* it was not ported correctly to D7, more details on a [related comment on d.o](https://www.drupal.org/node/1194274#comment-7738755),

I would anyway like to conserve this behaviour on the widget logic, I do not see it much different than widget template.
Do not worry about the presentation of that template for now.
This is more about preparing the base class to be able to use it.